# 槽号
int32 pot_id
# 数量
int32 job_nbs
# 槽盖控制
bool is_lid_open
# 通电控制
bool is_power_on
# 通电时间
int32 work_time
---
bool check_flag